code/modules/projectiles/targeting.dm

```diff
@@ -217,7 +217,7 @@ mob/living/proc/Targeted(var/obj/item/weapon/gun/I) //Self explanitory.
 mob/living/proc/NotTargeted(var/obj/item/weapon/gun/I)
 	if(!I.silenced)
 		for(var/mob/living/M in viewers(src))
-			to_chat(M, 'sound/weapons/TargetOff.ogg')
+			M << 'sound/weapons/TargetOff.ogg'
```

Comment:
This is how it always was and I'm not gonna spend time fixing oldcode when I should be fixing chat.